haas/ext/switches/mock.py

```diff
@@ -32,17 +33,30 @@ class MockSwitch(Switch):
     It's implementation is connectionless, so it is it's own session object as
     suggested int the superclass's documentation.
     """
-    id = Column(Integer, ForeignKey('switch.id'), primary_key=True)
 
     api_name = 'http://schema.massopencloud.org/haas/v0/switches/mock'
 
     __mapper_args__ = {
         'polymorphic_identity': api_name,
     }
+    
+    id = Column(Integer, ForeignKey('switch.id'), primary_key=True)
+    hostname = Column(String, nullable=False)
+    username = Column(String, nullable=False)
+    password = Column(String, nullable=False)
 
     @staticmethod
     def validate(kwargs):
-        Schema({}).validate(kwargs)
+        schema.Schema({
+            'username': basestring,
+            'hostname': basestring,
+            'password': basestring,
+        }).validate(kwargs)
+
+
+#    @staticmethod
+#    def validate(kwargs):
```

Comment:
I think we don't need to keep the old code commented out. It's in version control if we want to go back in time.